tensorflow_datasets/structured/movielens_utils.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2020 The TensorFlow Datasets Authors.
```

Comment:
Nit: I'd probably call this file `movielens_parsing.py`. Calling any file `utils` is basically giving up on meaningful naming :)

Reply:
Got it. I was under the impression that calling helper modules utils is an convention. I renamed it in 5462c161c468a2b675b154fc9fa169ae6c8af08c.